orchestrator/assets/construction.py

```diff
@@ -131,7 +131,7 @@ def dof_maintenance_cost(dhub: ResourceParam[DataHubResource]):
             "DOF Ops Costs Outside of Wos",
         ],
     )
-    df_out["fiscal_year"] = range(2019, 2024)
+    df_out["fiscal_year"] = range(2019, 2025)
```

Comment:
Great point. And this is a design decision. We can definitely use `datetime.datetime` to extract current year. However, this also depends on our update schedule. It's only our second year to caculate Construction Scope 3, we don't know on what schedule we'll rerun the pipeline. For example, if we use `current year + 1` here, if we rerun it in the early 2025, we won't have data from facilities probably till October, this line will break the pipe, with inconsistent length (fiscal_year is longer than data length.), since now we are only updating annually. I think it's OK to do this way, at least before we setup a formal update schedule.